tests/common/fixtures/advanced_reboot.py

```diff
@@ -127,10 +127,9 @@ def __buildTestbedData(self):
         self.rebootData['dut_username'] = secrets[invetory]['sonicadmin_user']
         self.rebootData['dut_password'] = secrets[invetory]['sonicadmin_password']
 
-        self.rebootData['default_ip_range'] = str(
-            ipaddress.ip_interface(self.mgFacts['minigraph_vlan_interfaces'][0]['addr'] + '/18').network
-        )
-
+        prefixLen = 18
+        testNetwork = ipaddress.ip_address(self.mgFacts['minigraph_vlan_interfaces'][0]['addr']) + (1 << (32 - prefixLen))
+        self.rebootData['default_ip_range'] = str(testNetwork) + '/{0}'.format(prefixLen)
```

Comment:
'testNetwork' doesn't give a network.  so the ip_range is going to be something like 192.168.64.1/18  instead of 192.168.64.0/18

Reply:
The code in the advanced reboot does [mask](https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/test/files/ptftests/advanced-reboot.py#L272) the hosts portion of the IP. So, this will not affect the IP generation. However to be complete, I will mask the host portion as well.